python/lsst/ts/ataos/ataos_csc.py

```diff
@@ -317,7 +326,11 @@ def mark_corrections(self, data, flag):
         if data.moveWhileExposing:
             self.move_while_exposing = flag
 
-        if data.all:
+        if getattr(data, "enableAll", False):
+            for key in self.corrections:
+                self.corrections[key] = flag
+            return
+        elif getattr(data, "disableAll", False):
```

Comment:
This is cleaner. But why do you need to use getattr instead of the more obvious `if data.enable/disableAll`? Please add a code comment explaining. I suspect I won't be the only reader of this code who wants to know.